idseq_pipeline/commands/non_host_alignment_functions.py

```diff
@@ -643,44 +634,34 @@ def run_chunk(part_suffix, remote_home_dir, remote_index_dir, remote_work_dir, r
         multihit_remote_outfile=multihit_remote_outfile if service == "gsnap" else multihit_remote_outfile[:-3]  # strip the .m8 for rapsearch as it adds that
     )
     if not lazy_run or not fetch_lazy_result(multihit_s3_outfile, multihit_local_outfile):
-        correct_number_of_output_columns = 12
-        min_column_number = 0
-        max_tries = 2
-        try_number = 1
-        while min_column_number != correct_number_of_output_columns and try_number <= max_tries:
+        @retry(2)
```

Comment:
Let's not apply the decorator here.   Keep it in mind for situations where it's easier to use.